external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaContinuousStream.scala

```diff
@@ -54,7 +55,8 @@ class KafkaContinuousStream(
   extends ContinuousStream with Logging {
 
   private val pollTimeoutMs = sourceOptions.getOrElse(CONSUMER_POLL_TIMEOUT, "512").toLong
-  private val includeHeaders = sourceOptions.getOrElse(INCLUDE_HEADERS, "false").toBoolean
+  private val includeHeaders =
```

Comment:
@HeartSaVioR I found why [the previous tests were failed](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/106646/testReport/), but the reason is a little bit wierd. It seems like the keys of `sourceOptions` are all lowercased, so the `includeHeaders` option is not detected without lowercased

Reply:
@dongjinleekr 
Yeah I also found missing spot on current code, raised #24942 . It would be nice to rebase and apply here as well if #24942 got merged earlier than this.